tools/pelican/content/ipynb/germany-compute-hospitalisierungsrate.ipynb

```diff
@@ -0,0 +1,4575 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #51.    # fig, ax = plot_hi(data["2021-10-01":])
</pre><p>I would clean this commented line up</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oscovida/oscovida/pull/282/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Also left this in as a quick way of trying the plot when developing it further. (See new entry in first cell of notebook.)</p>